templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -955,19 +955,19 @@ Resources:
               command: /opt/atlassian/bin/clone_deployment_repo
               ignoreErrors: true
             080_run_atl_init_node:
-              command: !Sub |
+              command:
                 cd /opt/atlassian/dc-deployments-automation/ && ./bin/install-ansible && ./bin/ansible-with-atl-env inv/aws_node_local aws_bitbucket_nfs_node.yml /var/log/ansible-bootstrap.log
               ignoreErrors: true
 
     Properties:
       BlockDeviceMappings:
         - DeviceName: /dev/sdf
           Ebs:
-            VolumeType: !If [IsHomeProvisionedIops, io1, gp2]
-            Iops: !If [IsHomeProvisionedIops, !Ref HomeIops, !Ref "AWS::NoValue"]
             DeleteOnTermination: !Ref HomeDeleteOnTermination
+            Iops: !If [IsHomeProvisionedIops, !Ref HomeIops, !Ref "AWS::NoValue"]
```

Comment:
This is the reason why we are ignoring `E2504` check for now. The rule currently cannot handle `Iops` and `VolumeType` when the condition is used for `If` function.

Reply:
Rough, can we raise an issue with the project?

Reply:
🖖, good thinking. I had it in my todo but it was too late yesterday to start raising issues in public projects.